### ISSUE TITLE
Network structure issues

### ISSUE THREAD

**Issue by dingzheng123** [tjqansthd/LapDepth-release#38]
Dear author, hello, I combined the overall network architecture diagram of your paper with this line of code dec2_up = self.decoder2_1(torch.cat([dec2, lap_lv5_up, rgb_lv4], dim=1)), and I have some questions. These are three feature concatenations. The number of channels of dec2 and lap_lv5_up are both 1, so how to concatenate them with the color difference image rgb_lv4, which has 3 channels? In the overall network structure, should L4 be placed in a 1*1 convolution, and then the dimension should be reduced together with layer2 before being concatenated with the result after upsampling with R5? I look forward to your reply. Thank you!
